cmd/explaintest/r/tpch.result

```diff
@@ -392,7 +392,7 @@ id	estRows	task	access object	operator info
 StreamAgg	1.00	root		funcs:sum(Column#20)->Column#18
 └─TableReader	1.00	root		data:StreamAgg
   └─StreamAgg	1.00	cop[tikv]		funcs:sum(mul(tpch.lineitem.l_extendedprice, tpch.lineitem.l_discount))->Column#20
-    └─Selection	3713857.91	cop[tikv]		ge(tpch.lineitem.l_discount, 0.05), ge(tpch.lineitem.l_shipdate, 1994-01-01 00:00:00.000000), le(tpch.lineitem.l_discount, 0.07), lt(tpch.lineitem.l_quantity, 24), lt(tpch.lineitem.l_shipdate, 1995-01-01 00:00:00.000000)
+    └─Selection	3713858.13	cop[tikv]		ge(tpch.lineitem.l_discount, 0.05), ge(tpch.lineitem.l_shipdate, 1994-01-01 00:00:00.000000), le(tpch.lineitem.l_discount, 0.07), lt(tpch.lineitem.l_quantity, 24), lt(tpch.lineitem.l_shipdate, 1995-01-01 00:00:00.000000)
```

Comment:
Why did this change? Caused by `cnt = clampRowCount(cnt, c.notNullCount())`?

Reply:
No. It's because the original `LessRowCountWithBktIdx` doesn't consider that a bucket could have the same lower and upper bound. Now in this PR, this case is considered.

I added the following code into the original `LessRowCountWithBktIdx` and the row count became the same as this PR.
```
if chunk.Compare(hg.Bounds.GetRow(bucketIdx*2+1), 0, &value) == 0 {
	return curCount - curRepeat, bucketIdx
}
```